lib/src/scope.dart

```diff
@@ -46,7 +46,7 @@ class Scope {
   /// Calls an isolate-scoped RPC named [method] with [params].
   ///
   /// This always adds the `isolateId` parameter to the RPC.
-  Future<Object> sendRequest(String method, [Map<String, Object> params]) async {
+  Future<dynamic> sendRequest(String method, [Map<String, Object> params]) async {
```

Comment:
Without it I get the following strong mode errors:

[error] The operator '[]' is not defined for the class 'Object'. (/google/src/cloud/dramos/drx6/google3/third_party/dart/vm_service_client/lib/src/frame.dart, line 67, col 19)
[error] The operator '[]' is not defined for the class 'Object'. (/google/src/cloud/dramos/drx6/google3/third_party/dart/vm_service_client/lib/src/frame.dart, line 71, col 63)
[error] The operator '[]' is not defined for the class 'Object'. (/google/src/cloud/dramos/drx6/google3/third_party/dart/vm_service_client/lib/src/scope.dart, line 39, col 15)
[error] The operator '[]' is not defined for the class 'Object'. (/google/src/cloud/dramos/drx6/google3/third_party/dart/vm_service_client/lib/src/scope.dart, line 64, col 19)
[error] The operator '[]' is not defined for the class 'Object'. (/google/src/cloud/dramos/drx6/google3/third_party/dart/vm_service_client/lib/src/scope.dart, line 72, col 63)